avm/res/azure-stack-hci/cluster/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,103 @@
+targetScope = 'subscription'
+
+metadata name = 'Deploy Azure Stack HCI Cluster in Azure with a 2 node switched configuration WAF aligned'
+metadata description = 'This test deploys an Azure VM to host a 2 node switched Azure Stack HCI cluster, validates the cluster configuration, and then deploys the cluster WAF aligned.'
+
+@description('Optional. The name of the Azure Stack HCI cluster - this must be a valid Active Directory computer name and will be the name of your cluster in Azure.')
+@maxLength(15)
+@minLength(4)
+param name string = 'hcicluster'
```

Comment:
Should ideally be a variable. The CI won't pass this value in

Reply:
resolved